docs/content/latest/secure/security-checklist.md

```diff
@@ -36,7 +36,7 @@ Run the YB-Master and YB-TServer services with a dedicated operating system user
 
 ### Restrict machine and port access
 
-Ensure that YugabyteDB runs in a trusted network environment.  Here are some steps to ensure that:
+Ensure that YugabyteDB runs in a trusted network environment, such that:
```

Comment:
Probably want to make this match the suggestion I made earlier, too

Reply:
fixed